examples/keras_io/tensorflow/vision/cutmix.py

```diff
@@ -48,10 +48,14 @@
 
 import numpy as np
 import pandas as pd
+import keras_core as keras
 import matplotlib.pyplot as plt
-import tensorflow as tf
+
 from keras_core import layers
-import keras_core as keras
+from tensorflow import clip_by_value
```

Comment:
Please add a comment explaining that the TF imports are for tf.data preprocessing